nilearn/glm/first_level/first_level.py

```diff
@@ -362,6 +362,9 @@ def fit(self, run_imgs, events=None, confounds=None,
             takes precedence over events and confounds.
 
         """
+        # Raise a warning if both design_matrices and confounds are provided
+        if design_matrices is not None and confounds is not None:
+            warn('If the design matrices are given, it takes precedence over events and confounds.')
```

Comment:
```suggestion
            warn('If design matrices are given, they take precedence over events and confounds.')
```

I'd lean toward using simpler words though. Perhaps `"If design matrices are supplied, confounds and events will be ignored."`.